src/dataset/breast_cancer.rs

```diff
@@ -67,17 +72,17 @@ pub fn load_dataset() -> Dataset<f32, f32> {
 mod tests {
 
     #[cfg(not(target_arch = "wasm32"))]
```

Comment:
I think that we need to remove this line too

Reply:
cool. I would like to reorganise the features this way:
* `default = []` or anything strictly necessary
* `serde = [default]` default plus anything for serialization
* `datasets = [serde]` serde plus datasets
* ....
how do you think we can do it?

Reply:
I think that for serde we don't need to specify `default` as a dependency

Reply:
ok. obviously once we have that `wasm` feature will only use default minus all the parts excluded using macros plus the JS part that is needed for bindgen.